test/test_tensordict.py

```diff
@@ -805,13 +784,15 @@ def test_zero_(self, td_name, device):
     def test_apply(self, td_name, device, inplace):
         td = getattr(self, td_name)(device)
         td_c = td.to_tensordict()
-        td_1 = td.apply(lambda x: x + 1, inplace=inplace)
+        td_1 = td.apply_(lambda x: x + 1, inplace=inplace)
```

Comment:
`apply` and `apply_` are two different methods, this test is for `apply` (which with `inplace=True` will call `apply_`). If we change `apply` to `apply_` then we are losing test coverage

Reply:
Reverted for now